### ISSUE TITLE
can not close tabs (Safari 15.0)

### ISSUE THREAD

**Issue by dcitguy2** [photopea/photopea#3759]
I was able to close tabs up until Safari 15 installed.  Those *&(^%) updates!

**Comment by photopea** [comment]
Hi, I just updated my MacBook to Safari 15.0, but I can still close Photopea tabs :(

What exactly should I do to not be able to close Photopea tabs in Safari 15?

**Comment by photopea** [comment]
Or what do you mean by "tabs"?

**Comment by dcitguy2** [comment]
well wouldn’t you know, when I went to the website to open a couple of tabs of two different pictures to edit, I tried to close them and presto, they closed.  I assure you, they did not close when I reported the bug.  This, I’m afraid, is the haphazard world of IT we live in.

here’s two open “tabs” so you know what I meant.

cheers!  BTW, I LOVE that you have this website - paying for Adobe monthly is a sin.
![Screen Shot 2021-09-23 at 16 05 49](https://user-images.githubusercontent.com/91242907/134481478-6cb35fe9-17c1-476b-9e78-5a8af419f101.jpg)
